api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/passwordless/account/PasswordlessAuthenticationLdapAccountsProperties.java

```diff
@@ -42,4 +42,10 @@ public class PasswordlessAuthenticationLdapAccountsProperties extends AbstractLd
      * indicates the user's name.
      */
     private String nameAttribute = "cn";
+
+    /**
+     * Name of the LDAP attribute that
```

Comment:
This is incorrect. This attribute does not disable passwordless functionality. It allows the passwordless flow to request a password prompt from user.

Reply:
My description was based on documentation :
https://apereo.github.io/cas/development/authentication/Passwordless-Authentication.html#disabling-passwordless-authentication-flow
should I say "disable passwordless authentication flow" ?

Reply:
You should say: "This setting allows the passwordless flow to request a password prompt from user."